app/templates/students.hbs

```diff
@@ -0,0 +1,25 @@
+{{page-title "Students"}}
+<div class="p-5">
+  <div class="mb-3">
+    <h2>Current courses:</h2>
+  </div>
+  <table class="table">
+    <thead>
+    <tr>
+      <th>Name</th>
+      <th>Last Name</th>
+      <th>Enrolled Course </th>
+    </tr>
+    </thead>
+    <tbody>
+    {{#each this.allStudents as |student|}}
+      <tr>
+        <td>{{student.name}}</td>
+        <td>{{student.lastName}}</td>
+        <td>{{'Empty'}}</td>
```

Comment:
This doesn't need to be like this, it could be just <td>Empty</td>, it's not wrong but just to let you know